syntax = "proto3";

package test.fuzz;

import "envoy/api/v2/core/base.proto";

import "google/protobuf/wrappers.proto";

// Common fuzzing input types.

message Header {
  string key = 1;
  string value = 2;
}

message Headers {
  repeated Header headers = 1;
}

message RequestInfo {
  envoy.api.v2.core.Metadata dynamic_metadata = 1;
  uint64 start_time = 2;
  google.protobuf.UInt32Value response_code = 3;
}
